src/main/java/jenkins/plugins/http_request/util/HttpRequestNameValuePair.java

```diff
@@ -4,20 +4,22 @@
 import hudson.model.AbstractDescribableImpl;
 import hudson.model.Descriptor;
 import hudson.util.FormValidation;
+import jenkins.plugins.http_request.HttpRequest;
+import org.apache.http.NameValuePair;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.QueryParameter;
 
 /**
  * @author Janario Oliveira
  */
-public class NameValuePair extends AbstractDescribableImpl<NameValuePair>
-        implements org.apache.http.NameValuePair {
+public class HttpRequestNameValuePair extends AbstractDescribableImpl<HttpRequestNameValuePair>
```

Comment:
Seems safe, since there was already an XStream alias so old `config.xml`s should have been using simply `<pair>`.